frontend/src/metabase/nav/components/DatabasePromptBanner/DatabasePromptBanner.tsx

```diff
@@ -27,12 +29,16 @@ export function DatabasePromptBanner({ location }: DatabasePromptBannerProps) {
     "/admin/databases/create",
   );
 
+  const adminEmail = MetabaseSettings.get("admin-email");
```

Comment:
When possible we use `useSelector(state => getSetting(state, key))` as `MetabaseSettings.get` is deprecated.

Other than this, LGTM